lib/shopify_api/plugs/admin_authenticator.ex

```diff
@@ -45,23 +45,31 @@ defmodule ShopifyAPI.Plugs.AdminAuthenticator do
 
   defp do_authentication(conn) do
     with {:ok, app} <- ConnHelpers.fetch_shopify_app(conn),
-         true <- ConnHelpers.verify_params_with_hmac(app, conn.query_params) do
+         {:hmac_verify, true} <-
+           {:hmac_verify, ConnHelpers.verify_params_with_hmac(app, conn.query_params)} do
```

Comment:
❤️ - might be able to keep this on one line if we `import ConnHelpers`? Up to you.

Reply:
I hummed and hawed on importing ConnHelpers, for some reason I have a dislike off doing import (maybe because it isn't obvious what methods your pulling in? maybe its just a silly fear!)

Reply:
I get that - in this case, maybe better to do an `import ConnHelpers, only: [fns]` thing, then?

Reply:
I went with the full import, since that helper is meant to extend the Plug.Conn functionality :D Also moved that return tuple in to a small private funct